webpack/assets/javascripts/react_app/common/MountingService.js

```diff
@@ -2,6 +2,8 @@ import React from 'react';
 import StatisticsChartsList from '../components/charts/StatisticsChartsList';
 import PowerStatus from '../components/hosts/powerStatus/';
 import NotificationContainer from '../components/notifications/';
+import ToastsList from
+'../components/toast/';
```

Comment:
maybe call it toastNotifications? I think people reading the code will not be sure what is the toast directory? :)